test/test_images.jl

```diff
@@ -4,12 +4,14 @@ using DataFrames
 import SloanDigitalSkySurvey: SDSS, WCSUtils, PSF
 
 using Celeste: Types, SampleData
-import Celeste: ModelInit, SkyImages, ElboDeriv
+import Celeste: ModelInit, SkyImages, ElboDeriv, Synthetic
 
 
 println("Running SkyImages tests.")
 
-field_dir = joinpath(dat_dir, "sample_field")
+const datadir = joinpath(Pkg.dir("Celeste"), "test", "data")
+
+field_dir = joinpath(datadir, "sample_field")
```

Comment:
There's no `sample_field` subdirectory in `test/data` anymore; the files are all in `test/data`.